overlord/snapstate/backend/mountunit.go

```diff
@@ -24,6 +24,11 @@ import (
 	"github.com/snapcore/snapd/progress"
 	"github.com/snapcore/snapd/snap"
 	"github.com/snapcore/snapd/systemd"
+	"github.com/snapcore/snapd/wrappers"
```

Comment:
I didn't notice originally that this used systemd directly, from that POV maybe it doesn't make sense to have the helper in wrappers, the code could just live here directly.

Reply:
OK, moved.